src/scipp/core/like.py

```diff
@@ -23,15 +23,26 @@ def zeros_like(
     var: _Union[_cpp.Variable,
                 _cpp.DataArray]) -> _Union[_cpp.Variable, _cpp.DataArray]:
     """
-    Constructs a new object with the same dims, shape, unit and dtype as the input
-    (:class:`Variable` or :class:`DataArray`), but with all values initialized to 0.
+    Return a Variable or DataArray with the same dims, shape, unit, and dtype as the
+    input and all values initialized to 0.
+
     If the input has variances, all variances in the output are set to 0.
-    If the input is a :class:`DataArray`, coordinates and attributes are shallow-copied
+    If the input is a data array, coordinates and attributes are shallow-copied
     and masks are deep copied.
 
-    :param var: Input variable or data array.
+    Parameters
+    ----------
+    var:
+        Input object defining dims, shape, unit, and dtype of the output
+
+    Returns
+    -------
+    :
+        New object of zeros.
 
-    :seealso: :py:func:`scipp.zeros` :py:func:`scipp.ones_like`
+    See Also
+    --------
+    zeros, ones_like : similar functionality
```

Comment:
The text after the colon is not shown in the rendered output. Do you know why?

Reply:
Were you not able to find a fix for this?

Reply:
I now tried some more formatting variations, and compared to how numpy writes docstrings, with no luck. It may be some interaction of the various extensions we use? Or a config flag I have not found yet.